File: 프로그래머스/lv0/120824. 짝수 홀수 개수/짝수 홀수 개수.py
def solution(num_list):
    
    even = 0
    odd = 0
    
    answer = []
    
    for i in num_list:
        if i % 2 == 0:
            even += 1
            answer.append(even)
        else:
            odd += 1
            answer.append(odd)

    return even, odd
    
    return answer

File: 프로그래머스/lv0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(num_list) {
    var answer = [0, 0];
    
    for (let a of num_list) {
        answer[a % 2] += 1
    }
    
    return answer;
    
}

// https://velog.io/@gagaeun/%ED%94%84%EB%A1%9C%EA%B7%B8%EB%9E%98%EB%A8%B8%EC%8A%A4-%EC%A7%9D%EC%88%98-%ED%99%80%EC%88%98-%EA%B0%9C%EC%88%98